<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeveloperMenu</class>
 <widget class="QWidget" name="DeveloperMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Developer Options</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>641</width>
     <height>381</height>
    </rect>
   </property>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>211</width>
      <height>40</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string>
      font-size: 24px;
      font-weight: bold;
      color: #2C3E50;
     </string>
    </property>
    <property name="text">
     <string>Developer Options</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="gameNameInput">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>500</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">   QLineEdit {
                    border-radius: 10px;
                    padding: 10px;
                    }</string>
    </property>
    <property name="placeholderText">
     <string>Enter game name</string>
    </property>
   </widget>
   <widget class="QComboBox" name="versionComboBox">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>120</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="visible">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
                    QComboBox {
                    border-radius: 10px;
                    padding: 10px;
                    }
                </string>
    </property>
    <item>
     <property name="text">
      <string>Pre_Alpha</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Alpha</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Beta</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Release Candidate</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>General Availability</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="createButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>
      QPushButton {
          background-color: #3498DB;
          color: white;
          border-radius: 20px;
          padding: 10px;
      }
      QPushButton:hover {
          background-color: #2980B9;
      }
     </string>
    </property>
    <property name="text">
     <string>Create Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="findButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>
      QPushButton {
          background-color: #E74C3C;
          color: white;
          border-radius: 20px;
          padding: 10px;
      }
      QPushButton:hover {
          background-color: #C0392B;
      }
     </string>
    </property>
    <property name="text">
     <string>Find Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="readGamesButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>240</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>
      QPushButton {
          background-color: #1ABC9C;
          color: white;
          border-radius: 20px;
          padding: 10px;
      }
      QPushButton:hover {
          background-color: #16A085;
      }
     </string>
    </property>
    <property name="text">
     <string>Read Games</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>
      QPushButton {
          background-color: #E74C3C;
          color: white;
          border-radius: 20px;
          padding: 10px;
      }
      QPushButton:hover {
          background-color: #C0392B;
      }
     </string>
    </property>
    <property name="text">
     <string>Delete Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>310</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>
      QPushButton {
          background-color: #3498DB;
          color: white;
          border-radius: 20px;
          padding: 10px;
      }
      QPushButton:hover {
          background-color: #2980B9;
      }
     </string>
    </property>
    <property name="text">
     <string>Update Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>
      QPushButton {
          background-color: #F39C12;
          color: white;
          border-radius: 20px;
          padding: 10px;
      }
      QPushButton:hover {
          background-color: #E67E22;
      }
     </string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="outputTextArea">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>631</width>
     <height>201</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
